helm/minio/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+description: High Performance, Kubernetes Native Object Storage
+name: minio
+version: 0.1.0
+appVersion: RELEASE.2021-06-14T01-29-23Z
```

Comment:
Can this chart be updated to handle embedded console deployment as well @nitisht

Reply:
yes, will make the change once we add it to opearot